src/storage/src/source/postgres/replication.rs

```diff
@@ -427,12 +427,6 @@ pub(crate) fn render<G: Scope<Timestamp = MzOffset>>(
             (*output_index, event.err_into())
         })
         .inner
-        .map_in_place(move |&mut ((_, ref data), _, ref mut diff)| {
-            // Ensure errors are never retracted.
-            if data.is_err() {
-                *diff = diff.abs();
```

Comment:
there's one more change on line 356 (can't comment on that diff) that doesn't rewind errors, we probably want to remove that too

Reply:
This is what I get trying to do two things at once. Good catch.